mesa_geo/geospace.py

```diff
@@ -95,7 +96,7 @@ def get_intersecting_agents(self, agent, other_agents=None):
         return intersecting_agents
 
     def get_neighbors_within_distance(
-        self, agent, distance, center=False, relation="intersects"
+            self, agent, distance, center=False, relation="intersects"
```

Comment:
This is a strange indentation, once again. You should check if it is possible to use Black in your IDE instead.

Reply:
It should be possible to use black, with some configurations. I'll look into it.